notebooks/13_Theta_phase_and_power.ipynb

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        theta_analytic_results.columns.get_loc("electrode {0}".format(electrode_id)) - 1
</pre><p>Could be an f-string :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
haha yes!